python/tvm/relax/frontend/nn/spec.py

```diff
@@ -59,6 +61,25 @@ def __repr__(self) -> str:
         return f"Tensor([{shape}], '{self.dtype}')"
 
 
+class Tuple:
+    """A tuple input or a list input"""
+
+    name: str
+    elements: Union[List[SpecAny], typing.Tuple[SpecAny, ...]]
```

Comment:
Is it the right way to represent variable-length tuples?

Reply:
you are right: https://stackoverflow.com/questions/54747253/how-to-annotate-function-that-takes-a-tuple-of-variable-length-variadic-tuple